modules/storages/app/contracts/storages/project_storages/base_contract.rb

```diff
@@ -49,7 +49,7 @@ class BaseContract < ::ModelContract
     attribute :project_folder_id
 
     attribute :project_folder_mode do
-      unless @model.project_folder_inactive? || @model.project_folder_manual?
+      unless @model.project_folder_inactive? || @model.project_folder_manual? || @model.project_folder_automatic?
```

Comment:
Will something like that work here?
```suggestion
      if Storages::ProjectStorage.project_folder_modes.keys.exclude?(@model.project_folder_mode)
```

Reply:
should work, replaced it.